apps/src/templates/curriculumCatalog/ExpandedCurriculumCatalogCard.jsx

```diff
@@ -15,33 +23,167 @@ const ExpandedCurriculumCatalogCard = ({
-            <BodyTwoText>{duration}</BodyTwoText>
+        <div className={style.flexDivider}>
+          <div className={style.courseOfferingContainer}>
+            <Heading3 style={{marginBottom: '8px'}}>
+              {courseDisplayName}
+            </Heading3>
+            <div className={style.infoContainer}>
+              <div className={style.iconWithDescription}>
+                <FontAwesome icon="user" className="fa-solid" />
+                <BodyTwoText>{gradeRange}</BodyTwoText>
+              </div>
+              <div className={style.iconWithDescription}>
+                <FontAwesome icon="clock" className="fa-solid" />
+                <BodyTwoText>{duration}</BodyTwoText>
+              </div>
+              <div className={style.iconWithDescription}>
+                <FontAwesome icon="book" className="fa-solid" />
+                <BodyTwoText className={style.subjectsText}>
+                  {i18n.topic() + ': ' + subjectsAndTopics.join(', ')}
+                </BodyTwoText>
+              </div>
+            </div>
+            <hr className={style.horizontalDivider} />
+            <div className={style.centerContentContainer}>
+              <div className={style.descriptionVideoContainer}>
+                <div className={style.descriptionContainer}>
+                  <BodyTwoText className={style.descriptionText}>
+                    {description}
+                  </BodyTwoText>
+                </div>
+                <div className={style.videoContainer}>
+                  <iframe
+                    width="100%"
+                    height="100%"
+                    style={{border: 'none'}}
+                    src={video}
+                    title=""
+                    allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share"
+                    allowFullScreen
+                  />
+                </div>
+              </div>
+              <div className={style.linksContainer}>
+                <div className={style.resourcesContainer}>
+                  <Heading6>{i18n.availableResources()}</Heading6>
+                  <hr className={style.thickDivider} />
+                  <TextLink text={i18n.lessonPlans()} href="#" />
+                  <hr className={style.horizontalDivider} />
+                  <TextLink text={i18n.slideDecks()} href="#" />
+                  <hr className={style.horizontalDivider} />
+                  <TextLink text={i18n.activityGuides()} href="#" />
+                  <hr className={style.horizontalDivider} />
+                  <TextLink text={i18n.answerKeysExemplars()} href="#" />
+                  <hr className={style.horizontalDivider} />
+                  <TextLink text={i18n.projectRubrics()} href="#" />
+                </div>
+                <div className={style.professionalLearningContainer}>
+                  <Heading6>{i18n.professionalLearning()}</Heading6>
```

Comment:
To follow accessibility guidelines in having headings go in descending order, could you make all of the `Heading6` tags into `Heading4` tags but then use a `visualAppearance` prop like shown here: https://github.com/code-dot-org/code-dot-org/tree/staging/apps/src/componentLibrary/typography (probably `visualAppearance="heading-sm"`?)